post_installation_configuration/cluster-tasks.adoc

```diff
@@ -508,6 +508,20 @@ You can create a `MachineSet` object to host only infrastructure components. You
 include::modules/infrastructure-components.adoc[leveloffset=+2]
 include::modules/nodes-scheduler-node-selectors-cluster.adoc[leveloffset=+2]
 
+[id="assigning-machine-set-resources-to-infra-nodes"]
+== Assigning machine set resources to infrastructure nodes
+
+After creating an infrastructure machine set, the `worker` and `infra` roles are applied to new infra nodes. Nodes with the `infra` role are not counted toward the total number of subscriptions that are required to run the environment, even when the `worker` role is also applied.
+
+However, with an infra node being assigned as a worker, there is a chance user workloads could get inadvertently assigned to an infra node. To avoid this, you can apply a taint to the infra node and tolerations for the pods you want to control.
+
+include::modules/binding-infra-node-workloads-using-taints-tolerations.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See xref:../nodes/scheduling/nodes-scheduler-about.adoc#nodes-scheduler-about[Controlling pod placement using the scheduler] for general information on scheduling a pod to a node.
+* See xref:moving-resources-to-infrastructure-machinesets[Moving resources to infrastructure machine sets] for instructions on scheduling pods to infra nodes.
```

Comment:
Maybe "...for instructions about scheduling..."

I can't find guidance in the ISG about this, but if you search the ISG for "about ", then you'll notice that most instances are "for information about..." and they conscientiously did not use "information on."

Reply:
It runs 3-2 in favor of _for information on_ in our docs, based on an Atom search. I will change it.